### ISSUE TITLE
Project Plan 

### ISSUE THREAD

**Issue by kdorheim** [ptrscll/hector-calibration-summer-2024#1]
@ptrscll could you share your pseudo document/project plan?

**Comment by ptrscll** [comment]
Yes! Here is the link: https://docs.google.com/document/d/1aZuFpbDJMYjNiykP4YAw1nfqc_PlRVTBDTFtfHucl-k/edit?usp=sharing
